### PR TITLE
Master Version of ToFlowSafe Version Compreet

### PR DESCRIPTION
**Pull request by JYoung413**
Same thing I did in toFlowModification branch but without a bunch of other files just things merged into master

### CODE REVIEW

File: src/com/nutrons/framework/util/FlowOperators.java [hunk 1]
```diff
@@ -11,14 +11,24 @@
...
-   * Generate a Flowable from a periodic call to a Supplier.
+   * Generate a Flowable from a periodic call to a Supplier. Safe Drops Backpressure
    *
    * @param ignored the number of time units to wait before calling the supplier again
    * @param <T> the type of the Flowable and Supplier
    */
   public static <T> Flowable<T> toFlow(Supplier<T> supplier, long ignored, TimeUnit unit) {
-    return Flowable.interval(ignored, unit, Schedulers.io())
-        .subscribeOn(Schedulers.io()).map((x) -> supplier.get());
+    return toFlowBackpressure(supplier, ignored, unit).onBackpressureDrop();
```
Comment (MiloDavis) [inline src/com/nutrons/framework/util/FlowOperators.java]: Why is this parameter called ignored?

File: src/com/nutrons/framework/util/FlowOperators.java [hunk 2]
```diff
@@ -11,14 +11,24 @@
...
+  }
+
+  /**
+   * Generate a Flowable from a periodic call to a Supplier. Does NOT drop Backpressure, Beware of overflow!
+   *
+   * @param ignored the number of time units to wait before calling the supplier again
+   * @param <T> the type of the Flowable and Supplier
+   */
+  public static <T> Flowable<T> toFlowBackpressure(Supplier<T> supplier, long ignored, TimeUnit unit) {
```
Comment (MiloDavis) [inline src/com/nutrons/framework/util/FlowOperators.java]: `ignored` is still a weird parameter name

File: src/com/nutrons/framework/util/FlowOperators.java [hunk 3]
```diff
@@ -37,11 +47,15 @@
    * @return the motor input values with values smaller than 0.2 removed
    */
   public static Flowable<Double> deadband(Flowable<Double> input) {
-    return input.map((x) -> abs(x) < 0.2 ? 0.0 : x);
+    return input.map((x) -> abs(x) < 0.4 ? 0.0 : x);
+  }
+
+  public static io.reactivex.functions.Function<Double, Double> deadbandMap(double minimum, double maximum, double remap) {
```
Comment (MiloDavis) [inline src/com/nutrons/framework/util/FlowOperators.java]: Use an import here

File: src/com/nutrons/framework/util/FlowOperators.java [hunk 4]
```diff
@@ -37,11 +47,15 @@
...
+    return bandMap(minimum, maximum, x -> remap);
   }
 
-  public static Function<Double, Double> deadbandMap(Flowable<Double> input) {
-    return x -> abs(x) < 0.2 ? 0.0 : x;
+  public static io.reactivex.functions.Function<Double, Double> bandMap(double minimum, double maximum, Function<Double, Double> remap) {
```
Comment (MiloDavis) [inline src/com/nutrons/framework/util/FlowOperators.java]: We were supposed to change the interface of `deadband`.  Is this in a different branch?

**Review by MiloDavis — CHANGES_REQUESTED** [review]

**Comment by kreeve** [comment]
Is this needed anymore?

**Review by MiloDavis — APPROVED** [review]

**Pull request closed without merging**